packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -436,8 +453,18 @@ export class MetadataEditor extends ReactWidget {
 
   render(): React.ReactElement {
     const inputElements = [];
-    for (const schemaProperty in this.schema) {
-      inputElements.push(this.renderField(schemaProperty));
+    for (const category in this.categories) {
+      if (category !== '_noCategory') {
+        inputElements.push(
+          <h4 style={{ flexBasis: '100%', paddingBottom: '10px' }}>
+            {' '}
+            {category}{' '}
```

Comment:
Two questions: Are the spaces needed here? and if so why is this not all on one line?